app/src/services/chefs.ts

```diff
@@ -2,92 +2,38 @@
 import axios from 'axios';
 import config from 'config';
 
-import type { AxiosInstance, AxiosRequestConfig, InternalAxiosRequestConfig } from 'axios';
+import { getChefsApiKey } from '../components/utils';
+
+import type { AxiosInstance, AxiosRequestConfig } from 'axios';
 
 /**
  * @function chefsAxios
  * Returns an Axios instance for the CHEFS API
  * @param {AxiosRequestConfig} options Axios request config options
  * @returns {AxiosInstance} An axios instance
  */
-function chefsAxios(options: AxiosRequestConfig = {}): AxiosInstance {
-  const instance = axios.create({
+function chefsAxios(formId: string, options: AxiosRequestConfig = {}): AxiosInstance {
+  return axios.create({
     baseURL: config.get('frontend.chefs.apiPath'),
     timeout: 10000,
-    ...options
+    ...options,
+    auth: { username: formId, password: getChefsApiKey(formId) ?? '' }
```

Comment:
May be better to have the auth attribute declared before the final ...options - just in the event you want to directly override the auth value via AxiosRequestConfig in the service layer you would be able to do so - ordered the way it is right now it's not configurable.